src/pypromice/pipeline/L0toL1.py

```diff
@@ -67,27 +67,12 @@ def toL1(L0, vars_df, T_0=273.15, tilt_threshold=-100):
 
     ds['z_boom_u'] = ds['z_boom_u'] * ((ds['t_u_interp'] + T_0)/T_0)**0.5      # Adjust sonic ranger readings for sensitivity to air temperature
 
-    if ds['gps_lat'].dtype.kind == 'O':                                        # Decode and reformat GPS information
-        if 'NH' in ds['gps_lat'].dropna(dim='time').values[1]:
-            ds = decodeGPS(ds, ['gps_lat','gps_lon','gps_time'])
-        elif 'L' in ds['gps_lat'].dropna(dim='time').values[1]:
-            logger.info('Found L in GPS string')
-            ds = decodeGPS(ds, ['gps_lat','gps_lon','gps_time'])
-            for l in ['gps_lat', 'gps_lon']:
-                ds[l] = ds[l]/100000
-        else:
-            try:
-                ds = decodeGPS(ds, ['gps_lat','gps_lon','gps_time'])          # TODO this is a work around specifically for L0 RAW processing for THU_U. Find a way to make this slicker
-
-            except:
-                print('Invalid GPS type {ds["gps_lat"].dtype} for decoding')
-
-    for l in ['gps_lat', 'gps_lon', 'gps_alt','gps_time']:
-        ds[l] = _reformatArray(ds[l])
```

Comment:
Hi @PennyHow 
I have a problem on all my branches containing this PR.
Since the those two lines were removed, gps_alt is is not coerced to numeric anymore and it ends up as string later in the processing, causing problem.

If gps_alt or gps_time are not touched by the GPS module, can we remove them from the exclusion here:
https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/eb668ce63248c6e90f9ccb2c8095635d3536f50e/src/pypromice/pipeline/L0toL1.py#L37-L39

so that they get also converted to numeric.

Reply:
I'll put them back in, no problem. I'll open a PR now to do this.